runatlantis.io/docs/using-atlantis.md

```diff
@@ -167,6 +167,49 @@ atlantis import -d dir 'aws_instance.example["foo"]' i-1234567890abcdef0 -- -var
 ```
 If a flag is needed to be always appended, see [Custom Workflow Use Cases](custom-workflows.html#adding-extra-arguments-to-terraform-commands).
 
+---
+## atlantis state rm
+```bash
+atlantis state [options] rm ADDRESS... -- [terraform import flags]
+```
+### Explanation
+Runs `terraform state rm` that matches the directory/project/workspace.
+This command discards the terraform plan result. After an import and before an apply, another `atlantis plan` must be run again.
```

Comment:
Should there be another flag like `atlantis terraform state rm --auto-plan-on-success` ?

We spoke about it in another comment for `atlantis terraform import`.